libscpi/test/test_parser.c

```diff
@@ -461,6 +462,7 @@ static void testIEEE4882(void) {
     TEST_IEEE4882("STATus:OPERation:CONDition?\r\n", "0\r\n");
     TEST_IEEE4882_REG_SET(SCPI_REG_OPERC, 1);
     TEST_IEEE4882("STATus:OPERation:CONDition?\r\n", "1\r\n");
+    TEST_IEEE4882_REG(SCPI_REG_OPERC, 1);
```

Comment:
`TEST_IEEE4882` tests outer behavior of the library, so how it communicates

`TEST_IEEE4882_REG` tests inner behavior of the library, so if the register has really the expected value.

Reply:
Yea, I understand. When I removed it I was thinking that this line was setting the register, but then I realized that it was actually verifying that the value didn't change after being read by the SCPI query.